CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 25.01+ (???)
 ------------------------------------------------------------------------
 - Fix: [#2701] Town names are not always generated correctly.
+- Fix: [#2764] Incorrect items selected in scroll view widgets.
```

Comment:
Hmm, to me, this makes it sound like it _always_ happens. It only happens after certain operations on the window list, right?
```suggestion
- Fix: [#2764] Scroll view item selection is wrong after switching window focus.
```